src/status_im2/common/contact_list_item/view.cljs

```diff
@@ -47,6 +50,7 @@
             :color (colors/theme-colors colors/neutral-50 colors/neutral-40)}]
           [quo/checkbox
            {:default-checked?    @checked?
```

Comment:
i think no need to use :default-checked? because it will be replaced by :checked? anyway in the compoennt